### PR TITLE
chore: add feature request template

### PR DESCRIPTION
**Pull request by rozenmd**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: "\U0001F680 Feature Request"
+description: Have a suggestion for how to improve wrangler? Let us know!
+title: "\U0001F680 BUG:"
```
Comment (JacobMGEvans) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:3]: Title update
```suggestion
title: "\U0001F680  Feature Request:"
```

Comment (rozenmd) [reply .github/ISSUE_TEMPLATE/feature-request.yaml:3]: 🤦

**Review by JacobMGEvans — APPROVED** [review]
Looks good to me, the Feature Request template is pretty straightforward since it's mostly up to the author

**Comment by JacobMGEvans** [comment]
Preview for reference: 
Screen Shot 2022-05-24 at 9 40 01 AM

**Comment by rozenmd** [comment]
TIL you can preview these!

**Pull request merged**